Windows/W32Util/DarkMode.h

```diff
@@ -3,7 +3,8 @@
 #define WIN32_LEAN_AND_MEAN
 #define NOMINMAX
 #include <Windows.h>
-#include <CommCtrl.h>
+#include <windows.h>
```

Comment:
Why do we have `Windows.h` and `windows.h`?  Although `"Common/CommonWindows.h` is preferred.

-[Unknown]